samples/StaticHtmlComposites/package.json

```diff
@@ -20,7 +20,9 @@
   "license": "MIT",
   "dependencies": {
     "@azure/communication-react": "1.0.1-beta.2",
-    "@azure/communication-common": "1.0.0",
+    "@azure/communication-common": "1.1.0",
+    "@azure/communication-calling": "1.3.2-beta.1 || 1.3.2",
```

Comment:
Why do we need to add these deps explicitly?

Reply:
common needed a update for some of its child dependencies and calling and chat were moved out of being peer dependencies so they needed to be declared here too.

Reply:
This is correct. An oversight, all our samples need to declare chat/calling as deps.